module/Permits/src/Permits/Controller/PermitsController.php

```diff
@@ -189,4 +191,8 @@ private function referredFromGovUkPermits(MvcEvent $e): bool
 
         return in_array($referer->getUri(), $this->govUkReferrers);
     }
+
+    protected function checkForRedirect($lvaId) {
+        return null;
+    }
```

Comment:
Same as above, how come?

Reply:
A trait these controllers use call this method in some of the trait methods. Static analysis was complaining about them not existing on the Controller. Though i believe the way the trait is used in the scope of these controllers the methods will never be called, this cleaned up the SA Errors.